src/EntryPoints/Specials/SpecialApproverCategories.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace ProfessionalWiki\PageApprovals\EntryPoints\Specials;
+
+use MediaWiki\MediaWikiServices;
+use SpecialPage;
+use PermissionsError;
+use LightnCandy\LightnCandy;
+
+class SpecialApproverCategories extends SpecialPage {
+
+	public function __construct() {
+		parent::__construct( 'ApproverCategories' );
+	}
+
+	public function isListed(): bool {
+		return $this->isAdmin();
+	}
+
+	public function execute( $subPage ): void {
+		if ( !$this->isAdmin() ) {
+			throw new PermissionsError( 'sysop' );
+		}
+
+		$output = $this->getOutput();
+		$approvers = [
+			[
+				'id' => 1,
+				'name' => 'John Doe',
+				'categories' => [ 'Category A', 'Category B' ]
+			],
+			[
+				'id' => 2,
+				'name' => 'Jane Smith',
+				'categories' => [ 'Category B', 'Category C' ]
+			]
+		]; // Placeholder
+
+		$templatePath = __DIR__ . '/../../../templates/ApproverCategories.mustache';
+		$template = file_get_contents( $templatePath );
+
+		if ( $template === false ) {
+			throw new \RuntimeException( "Template file not found: $templatePath" );
+		}
+
+		$phpStr = LightnCandy::compile( $template, [
```

Comment:
LightnCandy supports looping through a nested Loop in Mustache with Template Renderer doesn't seem to.

A ton of stuff on this page is to suppress Stan and PSalm errors.

Reply:
You can run `make stan-baseline` (in the extension's folder in the container) or exclude files via `phpstan.neon`. We have examples in some of our projects.